ChatRPG/Pages/UserCampaignOverview.razor

```diff
@@ -1,63 +1,75 @@
 @using ChatRPG.Data.Models
+@using ChatRPG.Services
 @using CampaignModel = ChatRPG.Data.Models.Campaign;
 
 <div class="container">
     @if (User is not null)
     {
+        <div class="dashboard-title-container">
+            <h1 class="title dashboard-title text-center">Dashboard</h1>
+        </div>
         <div class="row gap-3">
             <div class="col rounded custom-gray-container">
                 <div class="m-3">
-                    <h3>Your Campaigns</h3>
-                    <div class="vstack overflow-auto form-text-black" style="max-height: 400px;" id="scrollbar">
+                    <h3 class="text-center mb-4">Your Campaigns</h3>
+                    <div class="vstack overflow-auto form-text-black mb-1 scrollbar" style="max-height: 400px;" id="your-campaigns">
                         @foreach (CampaignModel campaign in Campaigns)
                         {
-                            <div class="card" style="width: 18rem;">
-                                <div class="card-body">
+                            <button class="card card-dim mb-2" @onclick='() => LaunchCampaign(campaign.Id)'>
+                                <div class="card-body col-12">
                                     <h5 class="card-title">@campaign.Title</h5>
-                                    <h6 class="card-subtitle mb-2 text-muted">@campaign.Characters.First(c => c.IsPlayer).Name</h6>
-                                    <p class="card-text text-truncate">@(campaign.CustomStartScenario ?? campaign.StartScenario?.Body ?? "No scenario")</p>
-                                    <p class="card-text">Started @campaign.StartedOn.ToShortDateString() @campaign.StartedOn.ToShortTimeString()</p>
-                                    <a href="/campaign/@campaign.Id" class="card-link">See more ></a>
+                                    <h6 class="card-subtitle mb-2 text-muted" style="font-size: 14px;">@campaign.Characters.First(c => c.IsPlayer).Name</h6>
+                                    <hr/>
+                                    <p class="card-text line-clamp text-start">@(campaign.CustomStartScenario ?? campaign.StartScenario?.Body ?? "No scenario")</p>
+                                    <p class="card-text" style="font-size: 14px;">Started @campaign.StartedOn.ToShortDateString() @campaign.StartedOn.ToShortTimeString()</p>
```

Comment:
Personligt synes jeg ikke tidspunktet er nødvendigt her, men hvis nogen føler strongly om det kan I bare resolve min kommentar. 
```suggestion
                                    <p class="card-text" style="font-size: 14px;">Started @campaign.StartedOn.ToShortDateString()</p>
```

Reply:
Jeg tilføjede det fordi jeg lavede flere campaigns på samme dag og synes at det manglede for at se forskel. Især hvis man bruger samme start scenario og campaign title, så er der ikke mange forskelle. Så kan det være rart med et timestamp.

Reply:
True, jeg tænkte på en mere generel use-case. Så er jeg enig i at vi kan have glæde af et timestamp.